scripts/decay-example.ipynb

```diff
@@ -0,0 +1,66 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "decay.plotdecay(100, 0.005, 1000)"
```

Comment:
Hi @ZoeRichter! This looks awesome! Can you please add axes labels to the plotdecay function? I think it will make the plot clearer.

Reply:
Sure, I can add them.